core/domain/email_manager.py

```diff
@@ -255,3 +297,57 @@ def send_moderator_action_email(
     _send_email(
         recipient_id, sender_id, intent, email_subject, full_email_content,
         bcc_admin=True)
+
+def send_exploration_membership_email(
```

Comment:
1) send_role_assignment_email might be a better name (we don't really have use the term 'members' for owners etc. of an exploration).

2) Nit: there should be two lines above top-level function definitions.